business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.exceptions import Warning as UserError
+from openerp.exceptions import ValidationError
+from openerp.tools.translate import _
+
+
+class BusinessRequirementResource(models.Model):
+    _name = "business.requirement.resource"
+    _description = "Business Requirement Resource"
+
+    sequence = fields.Integer('Sequence')
```

Comment:
FYI https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field
`If the technical name of the field (the variable name) is the same to the string of the label, don't put string parameter for new API fields, because it's automatically taken`